ui/objects/semantic-ui/js/objects/editform/components/BasicInfo.jsx

```diff
@@ -28,6 +32,33 @@ export const BasicInfo = ({ activeIndex, handleActive, record }) => {
     { value: "metr", text: "metr" },
     { value: "mm", text: "mm" },
   ];
+
+  const { formConfig } = useFormConfig();
+
+  console.log(formConfig)
+
+  const filteredVocabularies = (item) => {
+    const vocab = formConfig.vocabularies[item];
```

Comment:
This already implemented by `useVocabularyOptions` hook  from @js/oarepo-ui: https://github.com/oarepo/oarepo-ui/blob/main/oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/hooks.js#L56-L62